examples/linearAerodynamics/linearAerodynamics.m

```diff
@@ -66,16 +66,21 @@
 %md Two different types of elements are considered, node and frames. The nodes will be assigned in the first entry (index $1$) and the beam at the index $2$. The _elemType_ field is then:
 elements(1).elemType = 'node'  ;
 elements(2).elemType = 'frame' ;
-%md The node type has no cross-section geometry to assign (an empty array is automatically set). Since the frame element has no implemented a hollow cylindrical cross-section, then a `'generic'` cross-section (in $y$ and $z$) is used. Thus the _elemCrossSecParams_ field is:
-elements(2).elemCrossSecParams{1,1} = 'generic' ;
-elements(2).elemCrossSecParams{2,1} = [ A J Iyy Izz ] ;
+%md The node type has no cross-section geometry to assign (an empty array is automatically set). Since the frame element has a hollow cylindrical cross-section, then a `'pipe'` cross-section (in $y$ and $z$) is used. Thus the _elemCrossSecParams_ field is:
+elements(2).elemCrossSecParams{1,1} = 'pipe' ;
+elements(2).elemCrossSecParams{2,1} = [ dext dint ] ;
 %md Now the parameters to include aerodynamic forces automatically on the frame element are defined. The drag lift, and moment cross section functions are set in concordance with the function names located at the same example folder. Thus the _aeroCoefs_ field is a row cell defined as:
-elements(2).aeroCoefs   = {'dragCoefFunctionLA'; 'liftCoefFunctionLA'; 'momentCoefFunctionLA'} ;
+elements(2).chordVector = [0 dext 0]
+%md The geometrical non-linear effects are not considered in this case to compute the aerodynamic force. As consequence the wind load forces are computed on the reference configuration, and remains constant during the beam deformation. The field  aeroNumericalParams into  `elements` struct is then set to:
+numGaussPoints         = 4    ;
+computeAeroTangMatrix  = false;
+geometricNonLinearAero = false;
+elements(2).aeroNumericalParams = {numGaussPoints, computeAeroTangMatrix, geometricNonLinearAero}
```

Comment:
```suggestion
elements(2).aeroNumericalParams = {4, false, false}
```
the meaning of each entry should be already clear in the docs. i suggest removing lines 75 to 77. if wanted to explain more, add a comment